libs/application/templates/driving-license/src/forms/draft/subSectionTempInfo.ts

```diff
@@ -15,13 +15,13 @@ import {
   UserProfile,
 } from '../../types/schema'
 import { m } from '../../lib/messages'
-import { B_TEMP } from '../../shared/constants'
-import { isApplicationForCondition } from '../../lib/utils'
+import { B_TEMP, BE } from '../../lib/constants'
 
 export const subSectionTempInfo = buildSubSection({
   id: 'infoStep',
   title: m.informationApplicant,
-  condition: isApplicationForCondition(B_TEMP),
+  condition: (answers) =>
+    answers.applicationFor === B_TEMP || answers.applicationFor === BE,
```

Comment:
```suggestion
    [B_TEMP, BE].includes(answers.applicationFor)
```

Shorten everything :stuck_out_tongue_winking_eye: